tsdoc-config/.npmrc

```diff
@@ -0,0 +1,2 @@
+registry=https://registry.npmjs.org/

```

Comment:
Is this file necessary?

Reply:
This was just copied from the other projects in the repo.  Probably they should all be removed.  But let's deal with that in a separate PR.